treeherder/webapp/api/performance_data.py

```diff
@@ -296,8 +298,20 @@ class AlertPagination(pagination.CursorPagination):
     pagination_class = AlertPagination
 
     def update(self, request, *args, **kwargs):
-        request.data['classifier'] = request.user.username
-        return super(PerformanceAlertViewSet, self).update(request, *args, **kwargs)
+        data = request.data  # comment
```

Comment:
Perhaps the `# comment` could be replaced by renaming the `data` variable to make its purpose clearer?

Reply:
isn't the data naming part of how rest framework works? I don't think `# comment` is a very useful comment though :)

Here's a slightly more concise way of expressing what you're doing around here:

```py
from operator import itemgetter
(new_push_id, new_prev_push_id) = itemgetter('new_push_id', 'new_prev_push_id')(request.data)
```

(obv put the import at the top of the file if you decide to do this)